msi/release.sh

```diff
@@ -201,14 +201,30 @@ EOF
 # Make the .msi file
 mkdir -p obj &&
 wix/candle.exe -dVersion="${VERSION%%-*}" \
-    -arch $WIX_ARCH \
-    GitProduct.wxs GitComponents.wxs -o obj\\ \
-    -ext WixUtilExtension -ext WixTagExtension &&
+	-arch $WIX_ARCH \
+    GitProduct.wxs GitComponents.wxs \
+    GitChooseEnvironmentDlg.wxs GitChooseTerminalDlg.wxs \
+    GitCustomizeLineEndingsDlg.wxs GitLineEndingsDlg.wxs \
+    GitMaintenanceTypeDlg.wxs GitAdditionalOptionsDlg.wxs \
+    GitShellOptionsDlg.wxs \
+    -o obj\\ \
+    -ext WixUtilExtension -ext WixTagExtension \
+    -ext WixUIExtension &&
 wix/light.exe \
-    obj/GitProduct.wixobj \
+	obj/GitProduct.wixobj \
```

Comment:
Please keep the indentation as it was originally, and use tab indentation for any newly introduced code.